templates/partials/fsf_language.html

```diff
@@ -1,89 +1,110 @@
-<div class="u-fixed-width js-fsf-language-select" id="language">
-  <div class="p-fluid-grid--centered">
-    <a class="p-logo-link p-fluid-grid__item--small p-card p-flow-link" href="/first-snap/python" data-flow-link="python">
+<div class="row js-fsf-language-select" id="language">
+  <div class="col-small-2 col-medium-3 col-2 col-lang-select u-align--center">
```

Comment:
Because the `u-align--center` affects things inside the logo card, maybe we can just add centering to existing classes (.p-logo-link or .p-flow-link) instead of adding utility class?

Reply:
Doesn't need to be aligned center anyway... I seemed to make that up.